doc/radosgw/s3_objects_dedup.rst

```diff
@@ -34,13 +32,17 @@ Skipped Objects
 ***************
 Dedup Estimate process skips the following objects:
 
-- Objects smaller than 4 MB (unless they are multipart).
+- Objects smaller than rgw_dedup_min_obj_size_for_dedup (unless they are multipart).
```

Comment:
Suggest to either use double backticks around the config parameter or make it a `:confval:`.